x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/transport/CrossClusterApiKeySignatureManagerIntegTests.java

```diff
@@ -28,7 +30,7 @@ public class CrossClusterApiKeySignatureManagerIntegTests extends SecurityIntegT
     private static final String DYNAMIC_TEST_CLUSTER_ALIAS = "dynamic_test_cluster";
     private static final String STATIC_TEST_CLUSTER_ALIAS = "static_test_cluster";
 
-    public void testSignWithPemKeyConfig() {
+    public void testSignWithPemKeyConfig() throws GeneralSecurityException {
```

Comment:
Updated the verifier to move the auth error message building out of the verifier. That way the auth error can be constructed in a way that we can easily separate between cert chain validation error and signature verification error.